src/renderer/base/lib/base.vcxproj

```diff
@@ -11,6 +11,7 @@
   <Import Project="$(SolutionDir)src\common.build.pre.props" />
   <Import Project="$(SolutionDir)src\common.nugetversions.props" />
   <ItemGroup>
+    <ClCompile Include="..\CSSLengthPercentage.cpp" />

```

Comment:
Do you need to add this to `sources`, or will it be okay to not link anything related to CSSLengthPercentage inside Windows?